profiles/base.yaml

```diff
@@ -1221,6 +1221,8 @@ forge_container:
 spell_cycle:
 
 restock_shop:
+#Almanac noun
+almanac: almanac
```

Comment:
almanac_noun: would be more fitting here

Reply:
I can update. just followed the textbook:

Reply:
Agreed 👍